model/subnet.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+import "strings"
+
+type Subnet struct {
```

Comment:
nit: Naming this `PrivateSubnet` would better differentiate this and the existing `Subnet`(which is meant to be "public subnet").